sdcm/sct_config.py

```diff
@@ -1073,6 +1073,28 @@ class SCTConfiguration(dict):
 
         dict(name="max_events_severities", env="SCT_MAX_EVENTS_SEVERITIES", type=str_or_list,
              help="Limit severity level for event types"),
+
+        dict(name="data_device", env="SCT_DATA_DEVICE", type=str,
+             help="""Define which device to use for scylla data. Default for gce is attached. Default fow aws is instance_store.
```

Comment:
s/fow/for/

Reply:
fixed

Reply:
still see it

Reply:
@roydahan yes, i forgot to push changes. Done